joomla/joomla-cms
# 4.1 dev fields plugin

Pull Request for Issue # .

### Summary of Changes
Adds new options to transition settings to set custom fields as "required", "has to be blank", "should (not) contain a specific value"


### Testing Instructions
1. Create Custom Fields
2. Create an Article
3. Check if you recieve an error message when:
- A field is set as required but is blank
- A field is set as "must be blank" but is filled
- A field should contain a set value but doesn't
- A field should not contain a specific value but does
